### PR TITLE
Updates to VLASIATOR_ARCH files

### PR DESCRIPTION
**Pull request by MiroPalmu**
Updated the instructions for fixing broken VLSV files for Ukko and Carrington make arches.

Also added make arch and script to install dependencies for Cubbli 20, which should work for any local machine but cubbli 20 is only one that is tested. 

In the script I chose to install the libs to folder located at root folder of Vlasiator even if in other arch files libraries are usually located at ~/libs because I don't like that scripts from git repo modify files outside of the repo and anyway I think there really small change of sharing these libraries with other programs so there is no benefit to install them to $(HOME)

### CODE REVIEW

File: .gitignore
Comment (ykempf) [inline .gitignore:14]: Better add a newline here.

File: MAKE/Makefile.cubbli20
Comment (ykempf) [inline MAKE/Makefile.cubbli20:36]: SILO can be removed altogether from here I think.

File: MAKE/configure_cubbli20.sh
Comment (ykempf) [inline MAKE/configure_cubbli20.sh:72]: Here too add a newline please.

**Review by ykempf — CHANGES_REQUESTED** [review]
Well now that there's also a new feature coming in I'll wait for @ursg's opinion before we merge.

**Review by ursg — APPROVED** [review]
Looks all reasonable. Not sure about that .gitignore addition, but otherwise w could merge this right away.

Watch out that the changes to the carrington makfiles might cause a merge conflict with #629, but that will be pretty easy to resolve, I suppose.

File: MAKE/Makefile.cubbli20
Comment (henry2004y) [reply MAKE/Makefile.cubbli20:6]: Miro used shell script for this, which I guess is more universal. I have some extra functionalities in my Python script like checking if any of the dependencies has been installed, etc. The tricky part is the validity of the download links.

It's great to have an automatic script for installation, especially locally.

Comment (MiroPalmu) [reply MAKE/Makefile.cubbli20:6]: I tried to keep the script simple and easy to read what it is going to do.

My thought process was to just put commands from [here](https://github.com/fmihpc/vlasiator/wiki/Installing-Vlasiator) to one script that just installs the libraries and nothing else. (Also fix some of the outdated links etc.)

File: .gitignore
Comment (MiroPalmu) [reply .gitignore]: I removed .gitignore from the PR

**Pull request merged**
